ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/accordion/v1/accordion/clientlibs/commons/js/common.js

```diff
@@ -0,0 +1,261 @@
+                }
+            }
+
+
+            /**
+             * Annotates the item and its internals with
+             * the necessary style and accessibility attributes to indicate it is expanded.
+             *
+             * @private
+             * @param {HTMLElement} item The item to annotate as expanded
+             */
+            _expandItem(item) {
+                var index = this._getCachedItems().indexOf(item);
+                if (index > -1) {
+                    item.setAttribute(this.constructor.dataAttributes.item.expanded, "");
+                    var button = this._getCachedButtons()[index];
+                    var panel = this._getCachedPanels()[index];
+                    button.classList.add(this.constructor.cssClasses.button.expanded);
+                    // used to fix some known screen readers issues in reading the correct state of the 'aria-expanded' attribute
+                    // e.g. https://bugs.webkit.org/show_bug.cgi?id=210934
+                    setTimeout(function () {
+                        button.setAttribute("aria-expanded", true);
+                    }, this.constructor.delay);
+                    panel.classList.add(this.constructor.cssClasses.panel.expanded);
+                    panel.classList.remove(this.constructor.cssClasses.panel.hidden);
+                    panel.setAttribute("aria-hidden", false);
+                }
+            }
+
+            /**
+             * Annotates the item and its internals with
+             * the necessary style and accessibility attributes to indicate it is not expanded.
+             *
+             * @private
+             * @param {HTMLElement} item The item to annotate as not expanded
+             */
+            _collapseItem(item) {
+                var index = this._getCachedItems().indexOf(item);
+                if (index > -1) {
+                    item.removeAttribute(this.constructor.dataAttributes.item.expanded);
+                    var button = this._getCachedButtons()[index];
+                    var panel = this._getCachedPanels()[index];
+                    button.classList.remove(this.constructor.cssClasses.button.expanded);
+                    // used to fix some known screen readers issues in reading the correct state of the 'aria-expanded' attribute
+                    // e.g. https://bugs.webkit.org/show_bug.cgi?id=210934
+                    setTimeout(function () {
+                        button.setAttribute("aria-expanded", false);
+                    }, this.constructor.delay);
+                    panel.classList.add(this.constructor.cssClasses.panel.hidden);
+                    panel.classList.remove(this.constructor.cssClasses.panel.expanded);
+                    panel.setAttribute("aria-hidden", true);
+                }
+            }
+        }
+    }
+
+    window.CQ = window.CQ || {};
+    window.CQ.FormsCoreComponents = window.CQ.FormsCoreComponents || {};
```

Comment:
`window.CQ.FormsCoreComponents` we use this namespace only for authoring. We have not had use-cases of commons yet, so let's namespace this separately mentioning the commons.